design-library/source/_patterns/01-atoms/09-social-icons/social-icons.md

```diff
@@ -0,0 +1,6 @@
+# Atom: List Title
```

Comment:
Please update this file with the relevant info -- it's using "list title"

Reply:
Got it -- I had been neglecting the .md files entirely!